services/scabbard/libscabbard/src/migrations/diesel/postgres/migrations/2022-03-29-160500-create-scabbard-store-tables/down.sql

```diff
@@ -0,0 +1,28 @@
+-- Copyright 2018-2022 Cargill Incorporated
```

Comment:
This commit should be moved before the 42424f1384ad94feffd0dd20c83fac2902a84cff , as that commit's test fail without these migrations.

Reply:
moved